website/templates/include/profile/social.mako

```diff
@@ -4,14 +4,34 @@
 
         <form role="form" data-bind="submit: submit">
 
-            <div class="form-group">
-                <label>Personal Site</label>
-                <div class="input-group">
-                <span class="input-group-addon"><i class="glyphicon glyphicon-globe"></i></span>
-                <input class="form-control" data-bind="value: personal" placeholder="http://personalsite.com"/>
+            <label>Your Websites</label>
+            <div data-bind="sortable: {
+                        data: profileWebsites,
+                        options: {
+                            handle: '.sort-handle',
+                            containment: '#containDrag'
+                        }
+                    }">
+
+                <div>
+                    <div class="sort-handle">
+                        <i title="Click to Remove" class="btn text-danger pull-right  fa fa-times fa" data-bind="click: $parent.removeWebsite"></i>
+                        <div class="input-group" >
+                            <span class="input-group-addon"><i title="Drag to Reorder"  class="fa fa-bars"></i></span>
+                            <input class="form-control" data-bind="value: $parent.profileWebsites()[$index()]" placeholder="http://yourwebsite.com"/>
+                        </div>
+                    </div>
+                    <div class="form-group" data-bind="visible: $index() != ($parent.profileWebsites().length - 1)">
+                    </div>
                 </div>
             </div>
 
+            <div class="p-t-sm p-b-sm">
+                <a class="btn btn-default" data-bind="click: addWebsiteInput">
+                    Add Website
```

Comment:
To the first, I fixed the spelling.  Are you wanting the button disabled when you cannot add a website?  That was in before, but @sloria wanted it removed because of a conflict with a different part of the style guide I think.  Either works for me.

To the second, I am not seeing that terrible spacing on my browsers.  What browser are you running?